airflow/task/task_runner/standard_task_runner.py

```diff
@@ -82,6 +83,8 @@ def _start_by_fork(self):
                 args.func(args, dag=self.dag)
```

Comment:
Just wondering if we should flush() here too. If there are any calls to logger.error() in the task in this case I dont think they will get sent?

Reply:
Ya fair point, I'll make the change. Thanks!